.github/PULL_REQUEST_TEMPLATE/example.md

```diff
@@ -0,0 +1,11 @@
+# Ticket Number
+
+Create other templates and reference them with query params attached to URL creating PR.
```

Comment:
This seems to be just be an example file, can we remove it from this PR so it's not checked in as a real template?